code-reviews.md

```diff
@@ -65,6 +68,7 @@ accomplish this, everyone should follow these tips.
 - If discussions turn too philosophical or academic, move the discussion to a new
   issue or offline. In the meantime, the author has the final say on the current
   implementation.
+- Don't waste time [bike-shedding](https://en.wikipedia.org/wiki/Law_of_triviality).
```

Comment:
I appreciate the inclusion of this link, as I was previously unaware of the origin of "bike shed," but could it be worthwhile to avoid the abstraction?

Maybe something along the lines of:
"Don't waste time discussing trivial issues" 
_or_
 "Avoid focusing on trivial issues. This is often termed [bike-shedding](https://en.wikipedia.org/wiki/Law_of_triviality)."